adhocracy-plus/templates/account/signup.html

```diff
@@ -53,6 +53,7 @@ <h1>{% trans "Register" %}</h1>
         {% endif %}
 
         {% include 'a4_candy_contrib/includes/form_field.html' with field=form.captcha %}
+        {{ settings.a4_candy_cms_settings.OrganisationSettings.contacts | richtext }}
```

Comment:
@fuzzylogic2000 This is not ideal but other uses of settings links in help-text were all done in the html, I tried getting it into the modal help text but not very hard as I'm not sure how to get the correct string out of the rich-text to suit the sentence anyway so I though this was the best bet but pretty sure i'm wrong, or we make a setting just for support?
Maybe I can use `bleach` and only allow the link? that will only work if email is always added as a `mailto` link which probably isn't a good idea?! err brain is foggy sorry!